cli/copy_owner_windows.go

```diff
@@ -0,0 +1,44 @@
+// Copyright 2024 Northern.tech AS
+//
+//    Licensed under the Apache License, Version 2.0 (the "License");
+//    you may not use this file except in compliance with the License.
+//    You may obtain a copy of the License at
+//
+//        http://www.apache.org/licenses/LICENSE-2.0
+//
+//    Unless required by applicable law or agreed to in writing, software
+//    distributed under the License is distributed on an "AS IS" BASIS,
+//    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//    See the License for the specific language governing permissions and
+//    limitations under the License.
+
+//go:build windows
```

Comment:
Nice! One option could also be to have the files be `copy_unix` and `copy_windows`, and then ditch the build tags. But  it's up to you :)

Reply:
After a bit of reading, it seems like *_unix.go is not supported as a specific build file suffix. It attempts to build the _unix.go file for windows, so I think we need the build tags.

Reply:
Interesting, I had a look in the golang stdlib installed on my machine, and this is how they have them organised:

```
ls /usr/lib/go/src/os/path
path.go               path_test.go          path_windows.go
path_plan9.go         path_unix.go          path_windows_test.go
```

Reply:
According to this https://github.com/golang/go/blob/master/src/go/build/syslist.go unix is only used as a build tag, and not a filename. I think the path_unix.go has a build tag, and simply uses _unix.go to make it explicit.

Reply:
Alright, in any case. What you have done is just fine :)